website/app/templates/show.hbs

```diff
@@ -9,9 +9,19 @@
       @extra={{this.extra}}
     />
   {{/if}}
+
   <Doc::Page::Content @breakthrough={{not this.model.hasSidecar}} id={{this.model.showContentId}}>
     <DynamicTemplate @templateString={{this.renderedContent}} @componentId={{this.model.id}} />
+
+    {{#if this.relatedComponents}}
+      <hr />
+      <h2 class="doc-text-h2">Related</h2>
+      <Doc::Cards::Deck @cols="2" @layout="horizontal" @cards={{this.relatedComponents}} />
+
+      {{! <Doc::RelatedComponents /> }}
```

Comment:
I'm trying to add a new component but can't figure out how to get it to recognize it.

Reply:
@KristinLBradley the problem is in the path of the template file (`.hbs).
It's `app/components/doc/related-components/related-components.hbs` so the component invocation should be `<Doc::RelatedComponents::RelatedComponents />`.
What you want is `<Doc::RelatedComponents>` which means the file paths should be `app/components/doc/related-components/index.hbs` or `app/components/doc/related-components.hbs`

Reply:
Thanks!